share/completions/mtr.fish

```diff
@@ -0,0 +1,41 @@
+complete -c mtr -x -a "(__fish_print_hostnames)"
```

Comment:
Probably good to mention that this is a network diagnostic tool.

https://www.bitwizard.nl/mtr/ seems to be the website?